noisemodelling-emission/src/main/java/org/noise_planet/noisemodelling/emission/railway/nmpb/RailwayVehicleNMPBParameters.java

```diff
@@ -51,13 +50,13 @@ public RailwayVehicleNMPBParameters(String typeVehicle, double speedVehicle, dou
         setIdlingTime(idlingTime);
     }
 
-    public int getSpectreVer() {
+    /*public int getSpectreVer() {
```

Comment:
If this function is never used and you want to remove it, delete the code not comment it out.
Except for Java Getter/Setter, don't remove the functions even if it is not used (will be used outside of the library)